<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QWidget" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="nextTurnButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Закончить ход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="giveUpButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>450</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Сдаться</string>
   </property>
  </widget>
  <widget class="QLabel" name="gamerNumber">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Игрок 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="roundNumber">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Раунд 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="playerStates">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>50</y>
     <width>291</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bankStates">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>300</y>
     <width>291</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Купить сырье</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="buyMaterialsAmountChoice">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Продать ресурс</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sellResourcesAmountChoice">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Автоматизировать фабрику</string>
   </property>
  </widget>
  <widget class="QLabel" name="giveFactoryLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Подарить фабрику</string>
   </property>
  </widget>
  <widget class="QLabel" name="giveMoneyLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Подарить деньги</string>
   </property>
  </widget>
  <widget class="QLabel" name="giveMaterialsLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Подарить сырье</string>
   </property>
  </widget>
  <widget class="QLabel" name="giveResourcesLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Подарить ресурсы</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="giveFactoryChoice">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="giveMaterialsChoice">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="giveResourcesChoice">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>370</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Взять кредит</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="takeCreditChoice">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Взять страховку</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="buyMaterialsPriceChoice">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>50</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sellProductPriceChoice">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>90</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="giveMoneyChoice">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>290</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
